packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -42,15 +42,15 @@ export async function getDocumentsFromCQ({
 
     // We invoke the lambda that will start polling for the results
     // from the IHE Gateway and process them
-    await lambdaClient
+    lambdaClient
```

Comment:
removed the await because we dont want the api to wait for the lambda checking to see if all the results have come through from mirth

Reply:
But where's the error handling if we fail to call the lambda?